### PR TITLE
fix special chars in tags

### PR DESCRIPTION
**Pull request by iakov-aws**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: data-collection/deploy/module-organization.yaml [hunk 1]
```diff
@@ -208,19 +208,21 @@ Resources:
                   return path[::-1]
 
               @lru_cache(maxsize=10000)
-              def get_tags(self, id_):
+              def get_tags(self, id_, athena_friendly=False):
                   """returns a dict of tags"""
+                  def _process_key(key, athena_friendly):
+                      return key.replace(':', '_').replace('-', '_').replace(' ', '_').replace('.', '_') if athena_friendly else key
```
Comment (esc1144) [inline data-collection/deploy/module-organization.yaml:214]: Should we consider a regex and have we tested all the allowable characters in tags in Athena?
 . : + = @ _ / -

**Review by esc1144 — COMMENTED** [review]
We do something similar with the inventory module in which users can promote tags up as fields. We should revisit that and probably do the same escaping to _.

File: data-collection/deploy/module-organization.yaml (hunk 1 above)
Comment (iakov-aws) [reply data-collection/deploy/module-organization.yaml:214]: https://xkcd.com/1171/

Comment (esc1144) [reply data-collection/deploy/module-organization.yaml:214]: re.sub('[^a-zA-Z0-9]', '_', key)
Draconian but sure to be safe

Comment (iakov-aws) [reply data-collection/deploy/module-organization.yaml]: added regexp

**Review by esc1144 — APPROVED** [review]

**Pull request merged**